flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStDBMapKeyIterRequest.java

```diff
@@ -57,6 +57,7 @@ public Collection<UK> deserializeElement(List<RawEntry> entries, int userKeyOffs
             throws IOException {
         Collection<UK> deserializedEntries = new ArrayList<>(entries.size());
         for (RawEntry en : entries) {
+            // TODO: In Sync mode the user value can be null, so we need to skip null user value.
```

Comment:
how will this TODO be tracked - if this is not fixed in this PR then I suggest you track this in an issue an put the issue number here.